.travis.yml

```diff
@@ -28,6 +28,7 @@ addons:
 before_install:
     - cd
     - export INDEXES_PATH=/home/travis/build/indexes
+    - export NODE_ENV=test
```

Comment:
Well, you can put it in `env.global` or let it here. It is only used in `before_install` to start the data-system (it needs a writable path to write its index).